src/ResourceManager/Compute/Commands.Compute/StorageServices/SaveAzureVhdCommand.cs

```diff
@@ -0,0 +1,163 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Compute.Common;
+using Microsoft.Azure.Commands.Compute.Models;
+using Microsoft.Azure.Common.Authentication;
+using Microsoft.Azure.Common.Authentication.Models;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.WindowsAzure.Commands.Sync.Download;
+using System;
+using System.IO;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Compute.StorageServices
+{
+    [Cmdlet(VerbsData.Save, ProfileNouns.Vhd), OutputType(typeof(VhdDownloadContext))]
+    public class SaveAzureVhdCommand : ComputeClientBaseCmdlet
```

Comment:
Other class names for ARM cmdlets do not have 'Rm' pattern, either. (RemoveAzureVMCommand, NewAzureVMCommand, etc.)